packages/core/src/Keymap.ts

```diff
@@ -39,6 +40,13 @@ export class Keymap {
         });
     }
 
+    removeShortuct(patternString: string, priority = 0): void {
+        this._shortcutItems.push({
+            pattern: this._getPatternFromString(patternString),
+            priority,
```

Comment:
Wait, you can "remove a shortcut" with a "priority" ? What does that even mean ? I think that when someone wants to remove a shortcut by setting its `commandId` to `undefined`, he clearly wants the shortcut free of any command, no matter what.